.github/workflows/ci-aarchxx-cross.yml

```diff
@@ -180,10 +184,14 @@ jobs:
         username: ${{secrets.DYNAMORIO_NOTIFICATION_EMAIL_USERNAME}}
         password: ${{secrets.DYNAMORIO_NOTIFICATION_EMAIL_PASSWORD}}
         subject: |
-          [${{github.repository}}] ${{github.workflow}} FAILED
+          [Author Action Required] [${{github.repository}}] ${{github.workflow}} FAILED
           on ${{github.event_name}} at ${{github.ref}}
         body: |
           Github Actions CI workflow run FAILED!
+          Commit author must triage and fix the possible regression.
```

Comment:
This is a lot of duplication.  For duplication in the same file surely we can easily define a variable and use it here via `${{ myvar }}`.

For duplication across files: not sure how to do it: https://stackoverflow.com/questions/528281/how-can-i-include-a-yaml-file-inside-another implies it's hard.  But we're referencing `secrets` and other vars clearly coming from outside: so may be worth investigating.  Please look at https://docs.github.com/en/actions/sharing-automations/reusing-workflows and see if there's something there that could be easily done; if it's complex how about filing an issue and referencing in these files.

Reply:
Xref #7282 which refactors our GA configs to use reusable workflows. Will update this PR when that is merged.

Reply:
Updated.